src/main/java/fi/fmi/avi/converter/tac/AbstractTACSerializer.java

```diff
@@ -53,12 +49,32 @@ public TACTokenReconstructor removeReconstructor(final LexemeIdentity id) {
     @Override
     public abstract LexemeSequence tokenizeMessage(final AviationWeatherMessageOrCollection msg, final ConversionHints hints) throws SerializingException;
 
+    @Override
+    public ConversionResult<String> convertMessage(final AviationWeatherMessageOrCollection input, final ConversionHints hints) {
+        final ConversionResult<String> result = new ConversionResult<>();
```

Comment:
I wonder if the `input` type should be `S` like in the `AviMessageSpecificConverter` interface signature, for better type safety?